### ISSUE TITLE
Multiple Alerts on Website due to same reason

### ISSUE THREAD

**Issue by dneise** [fact-project/shifthelper#123]
I just got a couple of cals, since Main.js was not running. 
I think it is enough to show the Alert only once on the website ... 
![2016-10-23-212138_1920x1080_scrot](https://cloud.githubusercontent.com/assets/8200858/19628828/ecd0b176-9966-11e6-8d31-91a5c582e075.png)

**Comment by dneise** [comment]
do you disagree. Any opinion on this?

**Comment by MaxNoe** [comment]
I think I could do a groupby in the DB to show anf acknowledge all alerts of the same kind in one click.

**Comment by dneise** [comment]
Is showing only the last alert per check maybe enough?

**Comment by MaxNoe** [comment]
This semantically different, maybe yes.

**Comment by dneise** [comment]
Have a look at the page, whats the point in even showing all of them? Maybe one different alert hides inside many equal ones ... who knows ... 

![2016-11-03-114444_1920x1080_scrot](https://cloud.githubusercontent.com/assets/8200858/19963105/40f99324-a1bb-11e6-8a1e-d4bca6254605.png)

**Comment by dneise** [comment]
@MaxNoe can you remind me of the status of this? Can we maybe close this?

**Comment by MaxNoe** [comment]
As this is just cosmetics, I did not yet make any effort to tackle this.

I would leave it open, but this is just nice to have.
